src/agent.cpp

```diff
@@ -704,7 +774,8 @@ Agent::communicationImpl()
 {
     if ( M_communication )
     {
-        M_communication->execute( this );
+      // [Sanmit]: Turning this off since it adds default communication messages which can conflict with our comm messages.
```

Comment:
By default, the communication policy of Agent2D is round robin (i.e. players take turns who communicates). They communicate the location of the ball, players, and stamina. I'd expect that most of this information is redundant with full state turned on, but the only way to know for sure would be to test it.